system/Pager/PagerRenderer.php

```diff
@@ -25,14 +25,14 @@
 class PagerRenderer
 {
     /**
-     * First page number.
+     * First page number in the set of links to be displayed.
```

Comment:
You seemed to do something wrong with git.
Please get rid of unrelated commits from this PR.